### ISSUE TITLE
A11y problems in peer configuration screen

### ISSUE THREAD

**Issue by beqabeqa473** [donaldzou/WGDashboard#750]
**Describe The Problem**
In peer configuration screen, actions menu of peer as well as some action buttons inside that menu are not accessible for screenreaders.

**Expected Error / Traceback**
Buttons should contain descriptive names for screenreader

**To Reproduce**
Navigate to peer configuration screen with screenreader, for exampler with narrator or NVDA, a free open-source screenreader for windows

**OS Information:**
Ubuntu-24.04
python3.12

** Ways to fix:**
You could fix it yourself or i am ready to make a pull-request, but in this case i would like to ask, if it is allowed to put text on screen or it would be better to use aria-labels?

**Comment by donaldzou** [comment]
HI! Thank you so much for bringing this up, I will add aria labels to all icons in the upcoming release :) Honestly I never thought about this issue, thank you again.

**Comment by Danstiv** [comment]
> to all icons

Only for important icons please, that is, for icons on control elements.

**Comment by donaldzou** [comment]
Will do.


Best,

Donald Zou
________________________________

**Comment by DaanSelen** [comment]
I think this is ongoing @donaldzou so correct me if I am wrong.
